src/modules/keyboardmanager/common/LayoutMap.h

```diff
@@ -11,142 +12,19 @@ class LayoutMap
     // Stores mappings for all the virtual key codes to the name of the key
     std::map<DWORD, std::wstring> keyboardLayoutMap;
     std::mutex keyboardLayoutMap_mutex;
+    HKL previousLayout;
 
 public:
+    // Update Keyboard layout according to input locale identifier
+    void UpdateLayout();
+
     LayoutMap()
     {

```

Comment:
We should also initialize `previousLayout` here since there is a `if (layout == previousLayout)` check in `UpdateLayout`